tests/Mutant/Generator/MutationsGeneratorTest.php

```diff
@@ -28,7 +29,7 @@ public function test_it_collects_plus_mutation()
     {
         $codeCoverageDataMock = Mockery::mock(CodeCoverageData::class);
         $codeCoverageDataMock->shouldReceive('hasTestsOnLine')->andReturn(true);
-        $codeCoverageDataMock->shouldReceive('isLineFunctionSignature')->andReturn(false);
+        $codeCoverageDataMock->shouldReceive('hasExecutedMethodOnLine')->andReturn(false);
```

Comment:
As I see, the method `isLineFunctionSignature()` does not exists at all, and test is still green

Let's add `->once()` or appropriate method for all such cases (here and in other test cases) to ensure methods are executed needed times.

For this line it will be `$codeCoverageDataMock->shouldReceive('hasExecutedMethodOnLine')->once()->andReturn(false);`

Reply:
Done.